janusgraph-test/src/test/java/org/janusgraph/graphdb/serializer/SerializerGraphConfiguration.java

```diff
@@ -89,5 +93,34 @@ public void testOnlyRegisteredSerialization() {
         }
     }
 
+    @Test
+    public void testCustomHashMapSerialization() {
+        JanusGraphManagement mgmt = graph.openManagement();
+        PropertyKey time = mgmt.makePropertyKey("time").dataType(Integer.class).make();
```

Comment:
@sjudeng good catch, that unit test would be no good if it wasn't getting run. I converted it into one similar to the [SerializerTest#enumSerializeTest ](https://github.com/JanusGraph/janusgraph/blob/cb9c06aeb8ab95c3a4c4cc7575b7ad97ff79ddca/janusgraph-test/src/test/java/org/janusgraph/graphdb/serializer/SerializerTest.java#L225-L236) as suggested and removed the other one.